apps/web/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
```

Comment:
Why did this need to be added?

Reply:
maybe https://nextjs.org/docs/messages/failed-loading-swc

Reply:
web building is failing in docker env.

Reply:
I guess it is because SWC is a binary module specific to the platform and we don't install node modules in the container.